app/scrapers/__init__.py

```diff
@@ -34,7 +36,7 @@ def small_test():
 
 
 def feedgen(query, engine, count=10):
-    if engine == 'q':
+    if engine in ['q', 't']:
```

Comment:
Putting this logic in two different files can be the source of future bugs.  Please encapsulate this logic in one file or the other but not in both.

Reply:
@cclauss Please review again